### DISCUSSION TITLE
ogstun interface not seen while deploy the 5G SA in docker enviroment.

### DISCUSSION THREAD

**Discussion by Aniket621** [herlesupreeth/docker_open5gs#251] [General]
Hi all

I deployed 5G SA using below repositry (https://github.com/herlesupreeth/docker_open5gs) all containers showing up and NGAP interface and UE registered on network. But ogstun interface is not visible in core network and i am not able to ping the UE IP from Core Network. While check in host machine this script related on ogstun it is not present on this path /etc/systemd/network.

While deploy open5gs using package manager then it works fine and i am able to ping UE ip from both sides.

**Comment by herlesupreeth** [comment]
If you login to upf container you will find ogstun there.

```
docker exec -ti upf bash
ip a
```

**Comment by Aniket621** [comment]
In UPF container it is seen. One query route is available and also added NAT rule but still UE ip is not pinging

**Reply by herlesupreeth** [reply]
Can you please describe your setup and which UE are you using? And how are you trying to ping UE IP?

**Reply by Aniket621** [reply]
a) I am using Open5GS with UERANSIM (gNodeB+UE) and Open5gs is installed in VM1 with docker enviroment  and UERANSIM Installed in VM2. 
b) I am trying to ping the UE ip (192.168.100.X/24) from inside UPF Containers and my host machine.

**Reply by herlesupreeth** [reply]
Ah i see.. the reason you are not able to ping is maybe due to the NATing happening between gNB and 5GC. For now you can try running all in one VM. I will update the README over the weekend with instructions to run on different hosts

**Reply by Aniket621** [reply]
Hi @herlesupreeth hope u r doing good now i am able to ping the UE ip from Core network. But one question while i am running iperf command then No GTP tunnel is created in wireshark.

**Reply by herlesupreeth** [reply]
Irrespective of the traffic (ping or iperf) there should always be a GTP tunnel between gNB and 5GC to transport userplane traffic. While doing iperf ensure you are running iperf inside upf container and inside ueransim ue container.

**Comment by Aniket621** [comment]
Hi @herlesupreeth  I have one query is it mandatory to configure the subscriber in both Open5gs HSS and OsmoHLR

**Reply by herlesupreeth** [reply]
If you using only 5G SA then you can skip OsmoHLR. But you need to configure subscriber information in open5gs webui/mongodb
